### ISSUE TITLE
createAccount  error

### ISSUE THREAD

**Issue by walkero** [hyperledger/iroha#1208]
hi,
I use iroha-cli to test createAccount  function received error:
"Transaction has not passed stateless validation."

here is logs:

```
root@4572a6612c37:/opt/iroha_data# iroha-cli -account_name admin@test
Welcome to Iroha-Cli.
Choose what to do:
1. New transaction (tx)
2. New query (qry)
3. New transaction status request (st)
> : 1

Forming a new transactions, choose command to add:
1. Detach role from account (detach)
2. Add new role to account (apnd_role)
3. Create new role (crt_role)
4. Set account key/value detail (set_acc_kv)
5. Transfer Assets (tran_ast)
6. Grant permission over your account (grant_perm)
7. Subtract Assets Quantity from Account (sub_ast_qty)
8. Set Account Quorum (set_qrm)
9. Remove Signatory (rem_sign)
10. Create Domain (crt_dmn)
11. Revoke permission from account (revoke_perm)
12. Create Account (crt_acc)
13. Add Signatory to Account (add_sign)
14. Create Asset (crt_ast)
15. Add Peer to Iroha Network (add_peer)
16. Add Asset Quantity (add_ast_qty)
0. Back (b)
> : 12

Account Name: guest2
Domain Id: test
Public Key: 60b90a30bb0b31eb579cd4f4e08b71c641f1a3a53bbd8e35d61d654dfd6bd925
Command is formed. Choose what to do:
1. Add one more command to the transaction (add)
2. Send to Iroha peer (send)
3. Go back and start a new transaction (b)
4. Save as json file (save)
> : 2

Peer address (0.0.0.0): localhost
Peer port (50051): 50051
[2018-04-12 08:53:38.784550951][th:39866][info] TransactionResponseHandler Transaction successfully sent
Congratulation, your transaction was accepted for processing.
Its hash is dac86b41e126bcfdb99ed63131f8d09fbfbd8a4f7f86e673a842fa7d907cbadd
--------------------
Choose what to do:
1. New transaction (tx)
2. New query (qry)
3. New transaction status request (st)
> : 3

Choose action:
1. Get status of transaction (get_tx_info)
0. Back (b)
> : 1

Requested tx hash: dac86b41e126bcfdb99ed63131f8d09fbfbd8a4f7f86e673a842fa7d907cbadd
Tx hash is saved. Choose what to do:
1. Send to Iroha peer (send)
2. Save as json file (save)
0. Back (b)
> : 1

Peer address (0.0.0.0): localhost
Peer port (50051): 50051
Transaction has not passed stateless validation.
--------------------
Choose what to do:
1. New transaction (tx)
2. New query (qry)
3. New transaction status request (st)
```

How to solve this problem?

thanks

**Comment by walkero** [comment]
hi, @l4l  
yes, I pass the same data.

I found this is a bug for "Account name" at CreateAccount command/method,
I tried:
"Account name"  = guest2 is NG
"Account name"  = guestz is OK
"Account name"  = guest_z is NG
"Account name"  = guest_2 is NG

I read "6. Iroha API reference"
    http://iroha.readthedocs.io/en/latest/api/commands/#id14
"Account name" field, Constraint is [a-z_0-9]{1,32}
but it seems digit and "-" are not allowed.

thanks
